src/components/AppNavigation/AppNavigation.tsx

```diff
@@ -0,0 +1,37 @@
+import { Group, UnstyledButton } from '@mantine/core';
+import { FC, useCallback } from 'react';
+import { SupabaseService } from '../../services/supabase';
+
+export const AppNavigation: FC = () => {
+  const handleSignOut = useCallback(SupabaseService.signOut, []);
+
+  return (
+    <Group
+      sx={theme => ({
+        borderTop: `2px solid ${theme.colors.gray[3]}`,
+        justifyContent: 'space-around',
+      })}
+      px="xl"
+      py="lg"
+    >
+      <svg width="32" height="32" viewBox="0 0 33 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
width, height should be stored as a variable

Also, I think we should move out icons into standalone components